app/controllers/CreateGroupController.scala

```diff
@@ -224,7 +212,9 @@ class CreateGroupController @Inject()(
                     Ok(review_clients_to_add(clients, groupName, formWithErrors)).toFuture
                   }, (yes: Boolean) => {
                     if (yes)
-                      Redirect(controller.showSelectClients).toFuture
+                      clientService.clearSessionForSelectingClients().map(_ =>
```

Comment:
Should we rename this `clearSessionForFilteringClients` since this is the filter keys?

Reply:
yeah. probably